### DISCUSSION TITLE
No error output within the Logs for Invariant Test

### DISCUSSION THREAD

**Discussion by Naties29** [Cyfrin/security-and-auditing-full-course-s23#196] [General]
Hi Yall,

I am finishing up the debugging fuzzer class. But for some reason, my test outputs do not show the errors within the logs like they do for Patrick. 
```
Ran 1 test for test/unit/Invariant/Invariant.t.sol:Invariant
[FAIL. Reason: assertion failed: 29084802066 != 58169604132]
        [Sequence]
                sender=0x0000000000000000000000000000000000000A56 addr=[test/unit/Invariant/Handler.t.sol:Handler]0x5991A2dF15A8F6A256D3Ec51E99254Cd3fb576A9 calldata=swapPoolTokenForWethBasedOnOutputWeth(uint256) args=[1449877383 [1.449e9]]
 statefulFuzz_constantProductFormulaStaysTheSame() (runs: 256, calls: 8171, reverts: 0)
Logs:
  Bound result 1449877383
*AREA where Patricks Error is Displayed*
```
Within Patrick's output, underneath the Bound result, an error is displayed for him. I have made sure my foundry is the most up-to-date. My imports for both the handler and Invariant are the same as his in the video.

 I know that there is this error within my output, but it is still different to what Patrick's output is showing.
```
Reason: assertion failed: 29084802066 != 58169604132
```
Can anyone explain to me why this is occurring? Or has this been changed within a newer version of foundry?

Thanks yall!

**Comment by EngrPips** [comment]
Hello @Naties29, I can't explicitly say why your output result display format is different from Patrick's, but honestly, since it printed the appropriate output result, I would care less about the format, and software might behave slightly or totally differently from version to version

**Reply by Naties29** [reply]
Thanks @EngrPips , was just seeing if any one had an idea. I am assuming it must be a version difference to what he is using. No big deal ill work with what we have today lol! 

Thanks again for getting back to me! 

Regards,
Nate

**Reply by EngrPips** [reply]
Anytime, Friend. Keep crushing it.

**Comment by 0xMaxii** [comment]
I'm having the same issue, i don't know what to do to make the error in the terminal display logs.

**Reply by EngrPips** [reply]
Ho much verbosity did you specify?
